dolphinscheduler-api/src/main/resources/logback-spring.xml

```diff
@@ -61,4 +61,4 @@
         <appender-ref ref="STDOUT"/>
         <appender-ref ref="APILOGFILE"/>
     </root>
-</configuration>
+</configuration>
```

Comment:
Please remove unnessnary change.

Reply:
How should this be repaired? It seems that the file has not changed in the end.

Reply:
Oh, I see